BuildScript/BuildTargets/Testing.targets

```diff
@@ -115,9 +116,23 @@
     <!-- Run tests -->
 
     <PropertyGroup>
-      <_testName>$(_testAssemblyFileName).$(ConfigurationID).$(_platform).$(_executionRuntime).$(_targetRuntime)</_testName>
+      <_commaSeparatedTestCategoriesToExclude>$(TestCategoriesToExclude.Replace(';', ','))</_commaSeparatedTestCategoriesToExclude>
+      <_mergedTestCategoriesToExclude>$(_commaSeparatedTestCategoriesToExclude),$(_excludeCategoriesInTestConfiguration)</_mergedTestCategoriesToExclude>
+      <_excludeCategoriesCallArgumentString Condition="'$(_mergedTestCategoriesToExclude)' != ','">/exclude=$(_mergedTestCategoriesToExclude.Trim(','))</_excludeCategoriesCallArgumentString>
```

Comment:
for consistency & obviosuness reasons, we might want to do a second line as an alternate for this one with an explicitly empty argumentstring property. Otherwise you need to know that it's okay to reference an undeclared msbuild property and the value will be empty.